roles/lvm/tasks/volume.yml

```diff
@@ -9,6 +9,16 @@
     - docker
     - bootstrap
 
+- name: update lvg ansible module
+  connection: local
+  run_once: yes
+  get_url:
+    url: https://raw.githubusercontent.com/ansible/ansible-modules-extras/devel/system/lvg.py
+    dest: "{{ playbook_dir }}/library/lvg.py"
+  tags:
+    - docker
+    - bootstrap
```

Comment:
`bootstrap` tasks are for when machine are being bootstrapped with Packer. Could you remove this, please?